src/Domain/User.php

```diff
@@ -10,94 +10,88 @@
 /**
  * Domain model for User within the Baikal system
  */
-final class User
-{
+final class User extends DomainObject {
+
     /**
-     * @var Username
+     * Unique ID
+     *
+     * @var mixed
      */
-    private $username;
+    public $id;
```

Comment:
Why do you change these to public? do you have a use case for using properties directly?